templates/membership/join.html.twig

```diff
@@ -22,12 +23,19 @@
 {% block javascripts %}
     {% import 'javascript.js.twig' as js %}
     <script type="text/javascript">
-        Kernel.onLoad(function() {
-            App.runJoin('{{ form.vars.form_type_class|e('js') }}');
-            {{ js.address(form.address) }}
-        });
+        function initAutocomplete() {
+            autocomplete = new google.maps.places.Autocomplete(
+                /** @type {!HTMLInputElement} */(document.getElementById('autocomplete')),
+                {types: ['geocode']});
+            Kernel.onLoad(function() {
+                App.runJoin('{{ form.vars.form_type_class|e('js') }}', autocomplete);
+                {{ js.address(form.address) }}
+            });
+        }
     </script>
     <script src="https://www.google.com/recaptcha/api.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key={{ google_maps_api_key }}&amp;libraries=places&amp;callback=initAutocomplete"
```

Comment:
It looks like a security issue.
We should not exposed our secret in our sourcecode.
I saw that we did the same in many places in the code. Someone have an explanation ?
@ottaviano @illusionOfParadise @thomasbisignani @Remg  ?

Reply:
@B-Galati, to secure the API key @atamano should restrict access for this key on GCloud console if not done yet